src/store/modules/ui.js

```diff
@@ -3,20 +3,28 @@
 import * as getters from './getters/ui';
 import * as actions from './actions/ui';
 
-import { UI_TOGGLE_SLIDING_MENU, UI_SET_LOCALE } from '../mutation-types';
+import {
+  UI_TOGGLE_SLIDING_MENU,
+  UI_SET_LOCALE,
+  UI_SET_IS_HK,
+} from '../mutation-types';
 
 import { defaultLocale, availableLocales } from '../../locales';
 
 const initialState = () => ({
   locales: availableLocales,
   locale: defaultLocale,
+  isHK: true,
```

Comment:
how about storing location with getter isHK?

Reply:
but I have country and city judgement, in that case need to store 2 state

Reply:
i foresee there will be isTW isCN isJP... etc 
nonetheless this is ok for now